sql/hive/src/test/scala/org/apache/spark/sql/execution/benchmark/InsertIntoHiveTableBenchmark.scala

```diff
@@ -28,14 +28,11 @@ import org.apache.spark.sql.hive.test.TestHive
  * {{{
```

Comment:
I think we can now remove this comment https://github.com/apache/spark/commit/d7755cfd07554c132b7271730102b8b68eb56b28#diff-10e604a9a9d9c4bcc9cdc01049851095R170

and enable this test https://github.com/apache/spark/commit/d7755cfd07554c132b7271730102b8b68eb56b28#diff-10e604a9a9d9c4bcc9cdc01049851095R201. Feel free to ignore back if it fails for whatever reason.

Reply:
Looks we can just remove this comment too:

https://github.com/apache/spark/blob/6be8b935a4f7ce0dea2d7aaaf747c2e8e1a9f47a/sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala#L171-L173